_usage/limitations-of-babelfish.md

```diff
@@ -29,39 +29,39 @@ features are discovered, and additional features are added to MS SQL:
 | `CERTPROPERTY` function | This function is not supported. |
 | SQL keywords `CLUSTERED` and `NONCLUSTERED` for indexes and constraints | Babelfish accepts and ignores the `CLUSTERED` and `NONCLUSTERED` keywords. |
-| Collation, index on type dependant on the ICU library | An index on a user-defined type that depends on the ICU collation library (the library used by Babelfish) will not be invalidated when the version of the library is changed. For more information about collations, [see](/docs/usage/locales) |
+| Collation, index on type dependent on the ICU library | An index on a user-defined type that depends on the ICU collation library (the library used by Babelfish) will not be invalidated when the version of the library is changed. For more information about collations, [see](/docs/usage/locales) |
 | `COLLATIONPROPERTY` function | Collation properties are only implemented for the supported BBF collation types. For more information about collations [see](/docs/usage/locales) |
 | Column default | When creating a column default, the constraint name is ignored. To drop a column default, use the following syntax: `ALTER TABLE...ALTER COLUMN..DROP DEFAULT...` |
 | Column name: `IDENTITYCOL` | This column name is not supported. |
 | Column name: `$IDENTITY` | This column name is not supported. |
 | Column name: `$ROWGUID` | This column name is not supported. |
 | `COLUMNPROPERTY()` | This function is not supported. |
-| Blank column names with no column alias  | The `sqlcmd` and `psql` utilities handle columns with blank names in different ways. SQL Server `sqlcmd` returns a blank column name  PostgreSQL `psql` returns a generated column name. |
-| Column name case | Column names will be stored as lowercase in the PostgreSQL pg_attribute catalogs, but are stored in whatever case was specified in the `CREATE TABLE` statement in an internal Babelfish catalog. The `SELECT *` operation currently returns column names in lower case rather than in the case specified on the CREATE TABLE statement. This will be fixed in a future version of Babelfish, but until then a workaround is to either specify the columns explicitly in the SELECT statement, or to use SELECT * from a view.|
+| Blank column names with no column alias  | The `sqlcmd` and `psql` utilities handle columns with blank names in different ways. SQL Server `sqlcmd` returns a blank column name.  PostgreSQL `psql` returns a generated column name. |
+| Column name case | Column names will be stored as lowercase in the PostgreSQL `pg_attribute` catalog, but are stored in whatever case was specified in the `CREATE TABLE` statement in an internal Babelfish catalog. The `SELECT *` operation currently returns column names in lower case rather than in the case specified on the CREATE TABLE statement. This will be fixed in a future version of Babelfish, but until then a workaround is to either specify the columns explicitly in the `SELECT` statement, or to use `SELECT *` from a view.|
```

Comment:
The fix for the SELECT * limitation has recently been fixed, and as we discussed this should no longer be listed among our limitations:  "The `SELECT *` operation currently returns column names in lower case rather than in the case specified on the CREATE TABLE statement. This will be fixed in a future version of Babelfish, but until then a workaround is to either specify the columns explicitly in the `SELECT` statement, or to use `SELECT *` from a view."